tests/util/localregistry/localregistry.yaml

```diff
@@ -1,27 +1,24 @@
 # after creating the local registry, to push images to it you need to open up a port to the pod's registry:
-# $ POD=$(kubectl get pods --namespace kube-system -l k8s-app=kube-registry \
+# $ POD=$(kubectl get pods --namespace docker-registry -l k8s-app=kube-registry \
 #               -o template --template '{{range .items}}{{.metadata.name}} {{.status.phase}}{{"\n"}}{{end}}' \
 #               | grep Running | head -1 | cut -f1 -d' ')
-# $ kubectl port-forward --namespace kube-system $POD 5000:5000
+# $ kubectl port-forward --namespace docker-registry $POD 5000:5000
 
-apiVersion: v1
-kind: ReplicationController
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
Why ?

Reply:
Since I was using one pod only and needed that pod to stay I used a statefulset, but I realized ReplocationCOntroller or Deployment would have worked